src/components/RepoDetails/RepoDetail.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import star from "../../images/star.png";
+import styled from "styled-components";
+
+const Container = styled.div`
+  border: 1px solid;
+  display: flex;
+  flex-direction: row;
+  width: 70%;
+  padding: 1rem;
+  margin-bottom: -2px;
+`;
```

Comment:
border들이 겹치면서 2px로 나오는 경우가 있는데 이렇게 수정하면 좋을듯 합니다!
```suggestion
const Container = styled.div`
  border: 1px solid;
  display: flex;
  flex-direction: row;
  width: 70%;
  padding: 1rem;
  
  & + div {
    margin-top: -1px;
  }
`;
```

Reply:
<img width="1190" alt="스크린샷 2020-04-14 오후 9 51 40" src="https://user-images.githubusercontent.com/38802280/79227261-98181580-7e9a-11ea-8d8b-4ce7a7c77263.png">

<img width="1197" alt="스크린샷 2020-04-14 오후 9 54 27" src="https://user-images.githubusercontent.com/38802280/79227279-9c443300-7e9a-11ea-9b78-64ec06ad3c78.png">

Reply:
(사진으론 잘 안보이네요)

Reply:
넹! 말씀하신부분 참고해서 수정했어요~~👍